test/integration/client-side-operations-timeout/client_side_operations_timeout.prose.test.ts

```diff
@@ -901,4 +942,103 @@ describe('CSOT spec prose tests', function () {
       });
     });
   });
+
+  describe.skip(
```

Comment:
Can we put NODE-6403 as the `skipReason` here?

Reply:
done!